reinforcement_learning/common/misc.py

```diff
@@ -141,3 +141,6 @@ def get_execution_role(role_name="sagemaker", aws_account=None, aws_region=None)
     return client.get_role(RoleName=role_name)['Role']['Arn']
 
 
+def wait_for_training_job_to_complete(job_name):
```

Comment:
I am not quite sure if we need this function. I guess plotting should be able to done without waiting the training job to be finish?

Reply:
No, the function errors out for plotting if the job is not complete. Tested in all notebooks and the behaviour was pretty consistent.